force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -57,8 +67,19 @@ public with sharing class FlowControlLoopsCollections {
     public static Integer findMax(Integer num1, Integer num2) {
         // Initialize largestNum as null
         Integer largestNum = null;
-
-        return null; // Replace null with the variable you used to store the result
+           if (num1 == null || num2 == null) {
+            return null;
+           } else if (num1 == num2) {
+            largestNum = num1;
+            return largestNum;
```

Comment:
You can return your `largestNum` variable at the end after your ip statements because each check in your if is the same. Better to leave it all to the end and not have to write it 4 times separately within the if